### ISSUE TITLE
Broken links for data importing pages?

### ISSUE THREAD

**Issue by clibassi** [LOST-STATS/lost-stats.github.io#118]
I had on my list of additions I was planning to make to LOST some of the Stata code for importing various kinds of files, but the only importing page I can currently find is the [Import a Foreign Data File page](https://lost-stats.github.io/Other/import_a_foreign_data_file.html), and the "Also consider" links on that page seem to be broken. I may be missing them somewhere else, but just wanted to flag it.

**Comment by NickCH-K** [comment]
Yeah, those pages haven't been created yet, so the links will be broken until they are. They're currently on the [desired nonexistent pages](https://lost-stats.github.io/Desired_Nonexistent_Pages/desired_nonexistent_pages.html) page. So those would need to be added from scratch using the [New page template](https://github.com/LOST-STATS/LOST-STATS.github.io/blob/source/NewPageTemplate.md).
